### ISSUE TITLE
[1.16.1/1.16.2] SoundMuffler cannot be removed 

### ISSUE THREAD

**Issue by OverlordFreddo** [InventivetalentDev/SoundMuffler#29]
## What steps will reproduce the problem?  
1. Start Paper 1.16.1 server, load plugin
2. Craft and place SoundMuffler.
3. Attempt to remove SoundMuffler.

## What were you expecting to happen? What happened instead?  
Expected: SoundMuffler is destroyed/removed.
Actual: SoundMuffler still exists and entity can't be 'harmed'.

## What version of the plugin are you using? *Type /version &lt;Plugin Name&gt;*  
1.0.5-SNAPSHOT
PacketListenerAPI_v3.7.8-SNAPSHOT

## What Spigot version are you using? *Type /version*  
Paper-138 / 1.16.1
## What plugins are you using? *Type /plugins*  
Plugins (52): AdvancedGift, BottledExp, ChatPings*, ChestShop, ChestSort, CleanroomGenerator*, ColoredAnvils*, CoreProtect, CustomCrafting, CustomOreGen, Essentials, EssentialsSpawn, EXPBank-6.0.0, FastLeafDecay, Graves, Harbor, HeadsPlus, InteractionVisualizer, ItemInChatFree, Lasso, LastLoginAPI, LightAPI, LuckPerms, LWC, LWCTrust, mcMMO, MendingEnchant, MineableSpawners, MobsToEggs, Multiverse-Core, Multiverse-NetherPortals, Multiverse-Portals, OreAnnouncer, PerWorldInventory, PlaceholderAPI, Plan, PlugMan, ProtocolLib, PvPToggle, Sickle, TAB, TheNewEconomy, Towny, TownyChat, TownyNameUpdater, Vault, VeinMiner, VoidGenerator, WolfyUtilities, WorldBorder, WorldEdit, WorldGuard, SoundMuffler, PacketListenerAPI

## Do you have an error log? Use [pastebin.com](http://pastebin.com). *If you're not sure, upload your whole server log*  
No errors in log/console

## Did your client crash? *Upload errors in .minecraft/logs/latest.log as well*  
No client crash.

## Additional information?  *(Are you using Bungeecord? Did it work in previous versions? etc.)*
Not using Bungeecord.
Crafting and placing SoundMuffler works but not removing it.
Not tested in previous versions.

Workaround: used the `/minecraft:kill @e[distance=..10]` command in creative mode.

Please ask further questions at your leisure.  Any further testing would be done on a 1.16.2 production server as that's where the plugin is required.

**Comment by InventivetalentDev** [comment]
Please make sure you have the `soundmuffler.break` permission

Edit: also just realized the resource page doesn't list that, sorry about that!

**Comment by OverlordFreddo** [comment]
Does OP not include this permission?

Also I am happy to respond and test if the permission works, but the plugin isn't up to date. #30

**Comment by OverlordFreddo** [comment]
Since #30 has been resolved, I am now able to test the permissions and breaking of the muffler.

Added permissions `soundmuffler.place` and `soundmuffler.break`
```
08:16:56 OverlordFreddo issued server command: /lp applyedits zfAqyDUeZL
08:16:57 	[LP]: LOG > webeditor add soundmuffler.place true
		[LP]: LOG > webeditor add soundmuffler.break true
```
Then tested using LuckPerms Verbose:
```
08:31:36 Made OverlordFreddo no longer a server operator
08:31:46 CONSOLE issued server command: /give OverlordFreddo noteblock 1
			Giving 1 of note block to Freddo.
08:31:50 CONSOLE issued server command: /give OverlordFreddo whitewool 8
			Giving 8 of white wool to Freddo.
08:32:45 [LP]: Verbose logging enabled for checks matching OverlordFreddo & (soundmuffler.place | soundmuffler.break | soundmuffler).
08:33:25	OverlordFreddo lost connection: Disconnected
			-Freddo
08:33:27	UUID of player OverlordFreddo is [trimmed UUID]
			OverlordFreddo[/[trimmed] IP:50900]: logged in with entity id 3273 at ([res]315.15067823050794, 67.0, -71.0988979348973)
+Freddo
08:34:17	[LP]: VB > OverlordFreddo - soundmuffler.place - true
08:34:27	[LP]: Verbose logging disabled.
```
As you can see, a check was made for `soundmuffler.place` but not `soundmuffler.break` despite attempting both these actions.
Perhaps the permission `soundmuffler.craft` would be a good addition at the same time so that crafting can be restricted if required.


tl;dr still cannot break

**Comment by OverlordFreddo** [comment]
Also worth pointing out that the permissions list is not present in `plugin.yml` in the latest jar, therefore it didn't appear in LuckPerm's GUI.
(A full restart took place when installing the plugin)

**Comment by OverlordFreddo** [comment]
When I regain access to a server, I will update and try to recreate these
conditions.
